src/node/txdownloadman_impl.cpp

```diff
@@ -97,6 +97,7 @@ void TxDownloadManagerImpl::ActiveTipChange()
 
 void TxDownloadManagerImpl::BlockConnected(const std::shared_ptr<const CBlock>& pblock)
 {
+    // Orphanage may include transactions conflicted by this block.
```

Comment:
micro-nit: this being added in a32d396ef27d626e0c60e7ee8a74cbe3d918acb1 is weird